### ISSUE TITLE
Toggle button aria-label attribute value is not configurablbe

### ISSUE THREAD

**Issue by web-padawan** [vaadin/vaadin-select#218]
The `aria-label` currently used is hardcoded directly in template:

https://github.com/vaadin/vaadin-select/blob/81e0c22fe858d702b790e67cd08eb3a5b31f73d1/src/vaadin-select.html#L65

This is a violation of [AA Criteria of WCAG](https://www.w3.org/TR/WCAG21/#language-of-parts) if used on a page with lang != en. 

The value of the aria-label has to be configurable by the developer for i18n and accessibility.
